### DISCUSSION TITLE
Energy flowchart

### DISCUSSION THREAD

**Discussion by AditaNi** [Blackymas/NSPanel_HA_Blueprint#1505] [General]
Hi,

I recently discovered your work. Super work! 🤯
Is there anyway to put on a page  a Energy flowchart like from solar array to house and to different Lpads..like car, etc?

Regards
Adi

**Comment by edwardtfn** [comment]
We have started working at this and wanna finish this during this month: https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/168

**Reply by AditaNi** [reply]
Thanks for sharing this Info.
